### ISSUE TITLE
[Implementation]"Organic" Introduction of Search by DOM Elements + Addressing Min SDK Version Req

### ISSUE THREAD

**Issue by jas-kas** [getsentry/sentry#47181]
## TL;DR

This is an implementation ticket for [this design ticket](https://github.com/getsentry/sentry/issues/46601).

Please see the associated Figma [here](https://www.figma.com/file/sQEM8sWZKTbkZrFB33I1eY/Specs%3A-DOM-Search?node-id=37-2382&t=ZTgRQISFCDQVgKrB-0).

## Solution 

From the Figma page **Feedback 3**, here are the aspects we wish to implement and more context around them:

- Replace the placeholder message in the Search box [as seen in the Figma on the `Index/Default` image]
- `Index/AutoSuggestions/Key Selected/Concept B` (rather than Concept A) - this communicates to users in our autocomplete about the min SDK version requirement as well as highlights the new feature visually with the blue banner
-  `Index/Search Query Active/ Empty State` - provide a message in the Replay empty state when a user with an a project selected in the Page Filters has an older SDK (less than v7.44) attempts the `click` query. If multiple eligible projects are selected in the Page Filters, some with an older SDK, and some with an SDK version equal or greater to 7.44, don't show the message if there are replay results.

From the Figma page **Feedback 2**:

- Add the blue banner from `Index/Default` with a mention of the min SDK version. This banner should only appear if the project selected in the Page Filters has Replay set up.
